packages/core/.storybook/manager.js

```diff
@@ -1,7 +1,5 @@
 import { addons } from '@web/storybook-prebuilt/addons';
 import { create } from '@web/storybook-prebuilt/theming';
-import './public/common.css';
-import './public/manager.css';
```

Comment:
Do the styles still work in watch and the prod build of storybook after removing these? I don't remember if we dug into it.

Reply:
Yes, the styles still seem to work as far as I can tell.